.github/workflows/build_package-ri-cdktf-typescript.yml

```diff
@@ -1,4 +1,4 @@
-name: Build RI package for Terraform CDK
+name: Build RI package for CDK Terraform
```

Comment:
I thought the names are `Terraform CDK` (official) and `CDK TF` (short)

Reply:
though confusingly their repo is called `terraform-cdk`. I wish they stuck with that form for the written name. 
<img width="440" alt="image" src="https://user-images.githubusercontent.com/51779/213505083-65a0152d-0e5c-4865-bf5a-fc7760ad25a4.png">